### PR TITLE
marathon-swift 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by orta**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Adds Marathon: https://github.com/JohnSundell/Marathon

### CODE REVIEW

**Comment by orta** [comment]
Ah dang, I thought that linter error was just me setting up my local dev env wrong. Interesting, will discuss  with the maintainers about naming.

**Comment by orta** [comment]
Awesome, we're green 👍

File: Formula/marathon-swift.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class MarathonSwift < Formula
+  desc "Makes it easy to write, run and manage your Swift scripts 🏃"
+  homepage "https://github.com/JohnSundell/Marathon"
+  url "https://github.com/JohnSundell/Marathon/archive/1.0.1.tar.gz"
+  sha256 "647b098fc30af9d597d359dfb3e1aa2f62e29aa06aa4ffc6d417d1b9f9e54081"
+
+  depends_on :xcode => ["8.3", :build]
+
+  def install
+    ENV["SDKROOT"] = MacOS.sdk_path
```
Comment (ilovezfs) [inline Formula/marathon-swift.rb]: This is set automatically.

File: Formula/marathon-swift.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class MarathonSwift < Formula
+  desc "Makes it easy to write, run and manage your Swift scripts 🏃"
```
Comment (ilovezfs) [inline Formula/marathon-swift.rb]: Let's leave out the emoji

File: Formula/marathon-swift.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    if MacOS::Xcode.version >= "9.0"
+      system "swift", "package", "--disable-sandbox", "update"
+      system "swift", "build", "-c", "release", "-Xswiftc", "-static-stdlib", "--disable-sandbox"
+    else
+      ENV.delete("CC") # https://bugs.swift.org/browse/SR-3151
+      system "swift", "package", "update"
+      system "swift", "build", "-c", "release", "-Xswiftc", "-static-stdlib"
+    end
+
+    bin.install ".build/release/marathon"
```
Comment (ilovezfs) [inline Formula/marathon-swift.rb]: Please request a `make install` upstream.

File: Formula/marathon-swift.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  end
+
+  test do
+    system "#{bin}/marathon", "help"
```
Comment (ilovezfs) [inline Formula/marathon-swift.rb]: This needs to be a test that exercises and verifies the functionality of the software once installed.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Comment by orta** [comment]
@ilovezfs - do you know if there's anything unique about the Sierra and using the `install` command? I've  tried building out a make command for installing the app that calls:

```sh
PREFIX?=/usr/local
INSTALL_NAME = marathon

# [...]

install_bin:
	mkdir -p $(PREFIX)/bin
	mv .build/Release/Marathon .build/Release/$(INSTALL_NAME)
	/usr/bin/install -s -m 0755 .build/Release/$(INSTALL_NAME) $(PREFIX)/bin
```

But it only fails on Sierra, and when I tried using `cp` it was the same story. This is my first time making cask / using Makefiles for system setup, so am I missing something simple?

**Comment by orta** [comment]
@ilovezfs this is now green 👍

**Comment by orta** [comment]
Polite Poke

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew @orta! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by orta** [comment]
No - you rock - thanks!

**Comment by JohnSundell** [comment]
@orta @ilovezfs you both rock 🎸 Thanks a lot for this!

**Pull request closed without merging**